### DISCUSSION TITLE
Getting "Invalid authentication data connect time out" error while using token option from pycharm.

### DISCUSSION THREAD

**Discussion by dborker-cci** [community/community#54431] [New to GitHub]
I have been trying to push my code to the repo by signing in using a token, but I am getting errors.

I tried deleting the token and trying again. still, it's the same error.

![image](https://user-images.githubusercontent.com/125945399/236187555-4933ef82-42c0-4e02-af4f-ae46d46250cb.png)

**Comment by iamreiyn** [comment]
Hi @dborker-cci 

It's possible that a proxy is causing the problem. "Invalid authentication data" is a vague message that appears when a request to the GitHub API fails without a clear cause. "Connection reset" appears to be the actual message, which may be caused by a proxy or network firewall. Look at the error logs for additional information. Feel free to message if you need additional help, thank you.

**Reply by Micoder-dev** [reply]
Yeah i think this was the problem, while i am running the android studio in wifi router its working fine but at the same time if i try to use it my mobile hotspot the github only not working the remaining gradle sync are fine, if you have any solution plz update here

**Reply by oyehanif** [reply]
i am also facing same issue

**Reply by denis-lishchuk** [reply]
I have the same

**Reply by fahadali-87** [reply]
Facing the same issue. I'm not behind any proxy or firewall. Tried using my mobile data as well. Tried loggin in via Token and Login Via GitHub as well.

**Reply by fahadali-87** [reply]
> Facing the same issue. I'm not behind any proxy or firewall. Tried using my mobile data as well. Tried loggin in via Token and Login Via GitHub as well.

my issue got resolved. It was an update pending from WebStorm. Didn't get the notification from WebStorm, had to manually check for updates.

**Reply by tjader** [reply]
It's due to a security issue. You need to pull in the latest version of pycharm (2024.1.3).
https://blog.jetbrains.com/security/2024/06/updates-for-security-issue-affecting-intellij-based-ides-2023-1-and-github-plugin/?utm_source=product&utm_medium=link&utm_campaign=TBA

**Comment by Sony32FAT** [comment]
SSH

**Reply by Sony32FAT** [reply]
> SSH

Connect ### https://github.com/orgs/community/discussions/54431#discussioncomment-9781036

**Reply by Sony32FAT** [reply]
> > SSH
> 
> Connect ### https://github.com/orgs/community/discussions/54431#discussioncomment-9781036

Run

**Comment by ishubtripathi** [comment]
hii @dborker-cci 
follow this link to resolve your issue - https://www.youtube.com/watch?v=SCyx-x57xL0
it must solve your problem

**Comment by 1982moc** [comment]
updating helped.

**Reply by keikomori** [reply]
Thank you, this really helped me.

**Comment by minh-le1710** [comment]
Have you fixed this problem yet? I can help you
